timescaledb/tutorials/nfl-analytics/join-with-relational.md

```diff
@@ -0,0 +1,96 @@
+## Join time-series data with relational data for deeper analysis
+
+TimescaleDB is packaged as a PostgreSQL extension. As such, TimescaleDB is
+PostgreSQL with super-powers. You can do anything in TimescaleDB that you can 
+in PostgreSQL, including joining tables and combining data for further analysis.
+
+### The Mile-High advantage
+
+In professional football, it's a well-understood phenomenon that teams that
+visit Denver's Mile-High stadium are at a disadvantage because unlike the home team
+(the Denver Broncos), they are not accustomed to playing in high altitude.
+
+Earlier we ingested stadium data. Now we can run a query to see the performance of
+players at each position when they are playing at Mile High Stadium.
+
+Use this query to compare the average acceleration and yards run of individual players when they are performing in stadiums outside of Denver versus when they are playing in Denver. The columns `avg_acc_den` and `avg_yards_den` represent the acceleration and yard statistics while in Denver.  
```

Comment:
We may also need a similar bit of exposition to describe the data you're joining both tables on.